### PR TITLE
Added New Relic integration monitoring

### PR DESCRIPTION
**Pull request by pkudikyala**
**Description:**
Hi Team,

We have added our [Newrelic-RabbitMQ monitoring integration](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/rabbitmq-monitoring-integration/) link inside RabbitMQ monitoring documentation. Please review it and let us know if you need any further clarifications.

**For your reference please find the below screenshot:**

Screenshot 2022-08-05 at 4 51 27 PM

Thank you.

### CODE REVIEW

**Comment by michaelklishin** [comment]
Is the code for this integration open source? Can we make sure it does not cause extensive load of it uses the HTTP API, for example, query all queues to get one metric of just one queue? Some monitoring systems do that and I would rather have them excluded from the list.

**Comment by pkudikyala** [comment]
> Is the code for this integration open source? Can we make sure it does not cause extensive load of it uses the HTTP API, for example, query all queues to get one metric of just one queue? Some monitoring systems do that and I would rather have them excluded from the list.

Yes, New Relic is an open-source environment to monitor applications and servers.
The HTTP link redirects to the New Relic documentation and I'm sure there will be no extensive load on adding this HTTP link.
Kindly review and update me if there are any changes to be made.

**Comment by michaelklishin** [comment]
I have found evidence of this plugin using `/queues` and `/connections` which query for all entities:
 * https://github.com/newrelic/nri-rabbitmq/blob/73925c258149ee908b4fc61944b6e14f3d38a6fc/src/client/client.go#L24
 * https://github.com/newrelic/nri-rabbitmq/blob/73925c258149ee908b4fc61944b6e14f3d38a6fc/src/rabbitmq.go#L104
 * https://github.com/newrelic/nri-rabbitmq/blob/73925c258149ee908b4fc61944b6e14f3d38a6fc/src/client/client_test.go#L59

The biggest question is now how often this happens and whether this agent queries every node in the cluster.

File: site/monitoring.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Monitoring is a broad topic. This guide covers several :
 
  * [What is monitoring](#fundamentals), why it is important, what common approaches to it exist
  * Available monitoring options:
+     * [New Relic RabbitMQ](#with-newrelic) offers multiple dashboards for monitoring nodes
```
Comment (michaelklishin) [inline site/monitoring.md:33]: Sorry but listing New Relic here is clearly overstepping the line. We are listing
a number of general approaches (or available formats, if you will) to monitoring, and listing one vendor only makes sense to the marketing department of that vendor.

**Review by michaelklishin — COMMENTED** [review]
Sorry but listing New Relic as the first option in the overview section is clearly overstepping the line. 

We are listing
a number of general approaches (or available formats, if you will) to monitoring, and listing one vendor only makes sense to the marketing department of that vendor.

**Comment by michaelklishin** [comment]
To summarize: the PR as it is right now is a little insulting to the open source community because New Relic tried to list themselves as the first monitoring option in the list that
wasn't vendor-specific at all.

The New Relic agent uses the HTTP API and queries *all* queues, connections, etc at a configurable interval, which can be very expensive with a large number of entities. However,
that would be acceptable if only one cluster node was queried and not all of them.
This [seems to be the case](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/rabbitmq-monitoring-integration/#example-config) judging from the
configuration examples but I am not 100% certain my understanding is correct.

**Comment by lukebakken** [comment]
https://github.com/newrelic/nri-rabbitmq/issues/79

**Pull request closed without merging**